granulate_utils/linux/process.py

```diff
@@ -171,3 +171,33 @@ def search_for_process(filter: Callable[[psutil.Process], bool]) -> Iterator[psu
         with contextlib.suppress(NoSuchProcess, AccessDenied):
             if is_process_running(proc) and filter(proc):
                 yield proc
+
+
+class ProcCgroupLine:
```

Comment:
What's this? I don't see it previously removed in https://github.com/Granulate/granulate-utils/pull/257.